packages/one-app-server-bundler/__tests__/package.spec.js

```diff
@@ -12,15 +12,15 @@
  * under the License.
```

Comment:
Thanks GH for the interesting file transition `index-style-loader.js` to `package.spec.js`